pkgs/desktops/gnome-2/default.nix

```diff
@@ -62,7 +62,7 @@ lib.makeScope pkgs.newScope (self: with self; {
     autoreconfHook = pkgs.autoreconfHook269;
   };
 
-} // lib.optionalAttrs (config.allowAliases or true) {
+} // lib.optionalAttrs (config.allowAliases or false) {
```

Comment:
The usual interpretation is `or true` and I don't think users treat this as a ternary (`true`/`false`/"default") value and `or true` is important for the regular user experience.

I've opened https://github.com/NixOS/nixpkgs/pull/166792 to enforce proper usage of the aliases feature.

Reply:
nice, thanks. i'll close this pr